README.md

```diff
@@ -80,15 +80,15 @@ USAGE
 
 <!-- commands -->
 * [`mw app copy INSTALLATION-ID`](#mw-app-copy-installation-id)
+* [`mw app create node`](#mw-app-create-node)
+* [`mw app create php`](#mw-app-create-php)
```

Comment:
Is the renaming from `install` to `create` intentional? If so, I’d be interested in the reasoning, because this would introduce inconsistencies between these two apps and the other ones.